cdk.json

```diff
@@ -31,6 +31,13 @@
       "domain_name": "imap-mission.com",
       "region": "us-west-2"
     },
+    "efs_mount_config": {
+      "volume_name": "EFS",
+      "efs_path": "/data",
+      "mount_path": "/mnt/spice",
+      "efs_access_point_id": "efs-access-pt-id",
+      "efs_fs_id": "efs-filesystem-id"
+    },
```

Comment:
I don't think these need to be config parameters. We should probably just set them inside the code to fixed values because they won't be changing between the different accounts.

Reply:
I wasn't sure if I should do that because carrying that mount path consistent across different resources is important. If someone make changes to the EFS mount path but forgot to change it the in resources it was being used by, then that would cause issues. But I do see your point that these things won't change as much between different accounts. I will change those to fixed value for now and then we can revisit if that becomes an issue.

Reply:
What I more-so meant is that these should just be living within the code itself. So inside your stack you could define these: `efs_stack.mount_path`, and re-use them elsewhere. Then if the EFS gets updated or things get changed that stack should carry that info along rather than needing to update any configuration parameters.

Reply:
ok. I see. Yeah, I can totally do that.